### ISSUE TITLE
Should we create the branch book-v3 for the 3.0.0 release? 

### ISSUE THREAD

**Issue by camilamacedo86** [kubernetes-sigs/kubebuilder#1860]
**What do you want to happen?**

I'd like to reach a consensus definition if we should or not create he v3-book for he v3 release as it was done for v2? 

**Is there currently another issue associated with this?** 

https://github.com/kubernetes-sigs/kubebuilder/milestone/21

**Comment by camilamacedo86** [comment]
c/c @DirectXMan12 @estroz @Adirio

**Comment by Adirio** [comment]
I think so, yes.

We could also name it `book-latest` if it is not linking to any v3 domain directly so that we can keep using it when we move to v4 and so on.

**Comment by DirectXMan12** [comment]
FWIW, we have an alias for the main branch (current `master.book.kubebuilder.io` -- I keep meaning to fix the netlify config so that it recognizes our DNS entry for `nightly.book.kubebuilder.io`, but haven't had the time yet -- it'd be a change in netlify.toml to make it recognize the `nightly` subdomain)

**Comment by Adirio** [comment]
`master.*` and `nightly.*` sound ok, but the goal of having the `v2` branch was to keep the actual doc in synch with the last release.

**Comment by camilamacedo86** [comment]
Blocked by : https://github.com/kubernetes-sigs/kubebuilder/issues/1829. After we merge the migration guide for v3 I will create the branch.

**Comment by camilamacedo86** [comment]
Hi @Adirio, @estroz, @DirectXMan12, 

- [ ] - :book: update the docs over regards book-v3 branch #1960
- [ ] - Create the branch 
- [ ] - Configure netfly to work as: 

- `book-v3`: [book.kubebuilder.io](https://book.kubebuilder.io) -- current docs
- `book-v2`: [book-v2.kubebuilder.io](https://book.kubebuilder.io) -- legacy docs

@DirectXMan12 Could you help us to do the latest task/item here? I am not sure how/where we do that.

**Comment by Adirio** [comment]
I think that last change should wait until v3.0.0 (without any alpha) is released.

**Comment by camilamacedo86** [comment]
Hi @Adirio, 

I think this is the goal. We already released 2 nightly for the 3.0.0. IMO we are prepared dor that already. See the milestone 3.0.0. All that was planned is done.

**Comment by Adirio** [comment]
I wasn't clear enough. I meant literally after the release is made.
